generators/premiums/gpt3_restaurant_review/__init__.py

```diff
@@ -0,0 +1,48 @@
+import openai
```

Comment:
I think this shouldn't be here. Can you maybe remove this? Otherwise, everything else looks great. :)

Reply:
Yep, resolved that